src/netbase.cpp

```diff
@@ -61,31 +62,9 @@ std::string GetNetworkName(enum Network net) {
     }
 }
 
-bool static LookupIntern(const std::string& name, std::vector<CNetAddr>& vIP, unsigned int nMaxSolutions, bool fAllowLookup)
-{
-    vIP.clear();
-
-    if (!ValidAsCString(name)) {
-        return false;
-    }
-
-    {
-        CNetAddr addr;
-        // From our perspective, onion addresses are not hostnames but rather
-        // direct encodings of CNetAddr much like IPv4 dotted-decimal notation
-        // or IPv6 colon-separated hextet notation. Since we can't use
-        // getaddrinfo to decode them and it wouldn't make sense to resolve
-        // them, we return a network address representing it instead. See
-        // CNetAddr::SetSpecial(const std::string&) for more details.
-        if (addr.SetSpecial(name)) {
-            vIP.push_back(addr);
-            return true;
-        }
-    }
-
+static Optional<std::vector<CNetAddr>> LookupInternGetAddrInfo(const std::string& name, bool fAllowLookup) {
```

Comment:
This is changed in bfc1455 to `WrappedGetAddrInfo`, why not do that here?

Reply:
I could have done it already in the earlier commit. I suggest keeping the current commit order in not invalidate the ACK. Makes sense? :)

Thanks a lot for reviewing BTW ❤️

Reply:
Yeah that makes sense. Fuzzing is a labor of love